sass/layout/_layouts.scss

```diff
@@ -1,3 +1,21 @@
+.full {
+	width: 100%;
+}
+
+.clearfix:after {
+	content: "";
+	display: table;
+	clear: both;
+}
+
+.float-l {
+	float: left;
+}
+
+.float-r {
```

Comment:
.l-left and .l-right would probably make more sense since you are prefixing layout items with l-